src/check_ast.rs

```diff
@@ -1326,6 +1326,12 @@ where
                         self.locate_check(Range::from_located(expr)),
                     ));
                 }
+
+                if self.settings.enabled.contains(&CheckCode::B015) {
+                    if let Some(parent) = self.parents.last() {
```

Comment:
This incorrectly fires for any comparison expression *contained* in an expression statement (e.g. `print(x < y)`), even if it’s not the whole statement.

- #616

Reply:
Thanks, good call, that’s on me as reviewer.